src/main/kotlin/no/nav/syfo/personoppgavehendelse/PersonoppgavehendelseProducer.kt

```diff
@@ -42,4 +44,33 @@ class PersonoppgavehendelseProducer(
             throw e
         }
     }
+
+    fun sendPersonoppgavehendelse( // TODO: Merge the two functions in this class
```

Comment:
Denne likner veldig på funksjonen under så hadde vært fint om TODOen ble gjort

Reply:
Vi har en del TODOs som vi strør rundt oss for å slippe å gjøre alt på en gang. Vi tar en opprydning når dette er ute :)